src/lib/services/blockchainWatcher.ts

```diff
@@ -11,8 +11,8 @@
 //   switch (tokenType) {
 //     case 'DEMAND':
 //       return 'demand'
```

Comment:
why's this whole file commented out, did I miss something?

Reply:
It as originally there